server/schedule/operator/step.go

```diff
@@ -348,6 +384,11 @@ func (mr MergeRegion) Influence(opInfluence OpInfluence, region *core.RegionInfo
 	}
 }
 
+// Timeout returns true if the step is timeout.
+func (mr MergeRegion) Timeout(start time.Time, regionSize int64) bool {
+	return time.Since(start) > maxFastWaitTime(regionSize)*10
```

Comment:
Why * 10?

Reply:
It is one empirical value from test enviroment, the merge costs more time than `remove`,`transfer`.